fbw-a380x/src/systems/instruments/src/MFD/pages/FMS/MfdFmsPerf.tsx

```diff
@@ -2538,14 +2538,7 @@ export class MfdFmsPerf extends FmsPage<MfdFmsPerfProps> {
                       <RadioButtonGroup
                         values={['CONF 3', 'FULL']}
                         selectedIndex={this.apprSelectedFlapsIndex}
-                        onModified={(v) => {
-                          this.props.fmcService.master?.fmgc.data.approachFlapConfig.set(v + 3);
-                          if (v === 0) {
-                            SimVar.SetSimVarValue('L:A32NX_SPEEDS_LANDING_CONF3', 'boolean', true);
-                          } else {
-                            SimVar.SetSimVarValue('L:A32NX_SPEEDS_LANDING_CONF3', 'boolean', false);
-                          }
-                        }}
+                        onModified={(v) => this.props.fmcService.master?.fmgc.data.approachFlapConfig.set(v + 3)}
```

Comment:
```suggestion
                        onModified={(v) => this.props.fmcService.master?.fmgc.data.approachFlapConfig.set(v + 3)}
```
This kind of thing is a bug waiting to happen IMO. There should be an explicit mapping from the UI value to the enum approachFlagConfig expects.